docs/scripts_md/batch_uploads_tarchive.md

```diff
@@ -0,0 +1,58 @@
+# NAME
+
+batch\_uploads\_tarchive - upload a batch of tarchives using script tarchiveLoader
+
+# SYNOPSIS
+
+./batch\_uploads\_tarchive
+
+# DESCRIPTION
+
+This script uploads a list of tarchive files to the database by calling script 
+`tarchiveLoader` on each file in succession. The list of files to process is read 
+from STDIN, one file name per line. Each file name is assumed to be a path relative 
+to `tarchiveLibraryDir` (see below).
+
+The following settings of file `$ENV{LORIS_CONFIG}/.loris-mri/prod` affect the 
+behvavior of batch\_uploads\_tarchive (where `$ENV{LORIS_CONFIG}` is the value of
+the Unix environment variable LORIS\_CONFIG):
+
+- **dataDirBasepath** : controls where the STDOUT and STDERR of each qsub command
+  (see below) will go, namely in 
+  `$dataDirBasepath/batch_output/tarstdout.log<index>` and
+  `$dataDirBasepath/batch_output/tarstderr.log<index>`
+  (where `<index>` is the index of the tarchive file processed, the first file having
+   index 1).
+- **tarchiveLibraryDir**: directory that contains the tarchive files to process. The path
+  of the files listed on STDIN should be relative to this directory.
+- **is\_qsub**: whether the output (STDOUT) of each tarchiveLoader command should be
+  processed by the qsub Unix command (allows batch execution of jobs on the 
+  Sun Grid Engine, if available). If set, then the qsub command will send its STDOUT
+  and STDERR according to the value of dataDirBasepath (see above). 
+- **mail\_use**: upon completion of the script, an email will be sent to email address
+  $mail\_user cointaining the list of files processed by batch\_upload\_tarchive
+
+File prod should also contain the information needed to connect to the database in an
+array `@db` containing four elements:
+
+- The database name
+- The SQL user name used to connect ot the database
+- The password for the user identified above
+- The database hostname
+
```

Comment:
since the lines on the `@db` is applicable to all the scripts, should we put it everywhere?

Reply:
I suppose we could. Up to you. 😄